bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.ts

```diff
@@ -166,11 +167,24 @@ export class KtbEvaluationDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  @Input()
+  get showChart(): boolean {
+    return this._showChart;
+  }
+  set showChart(value: boolean) {
+    if (this._showChart !== value) {
+      this._showChart = value;
+      this._changeDetectorRef.markForCheck();
```

Comment:
A simple input might be sufficient here, it should do the same check internally.
```@Input() showChart: boolean;```